src/iotile/models/datafilterargs.ts

```diff
@@ -20,6 +21,9 @@ export class DataFilterArgs {
     if (this.page) {
       parameters.push('page=' + this.page);
     }
+    if (this.streamerId) {
```

Comment:
But please test this (manually on the browser), as I forgot if it is `streamer_id0` or `streamer_id_0`